js/researches/english/EnglishParticiple.js

```diff
@@ -0,0 +1,153 @@
+var matchDeterminers = require( "../english/passivevoice-english/wordIndicesRegexes.js" )().determiners;
+var matchHaving = require( "../english/passivevoice-english/wordIndicesRegexes.js" )().having;
+
+var irregularExclusionArray = [ "get", "gets", "getting", "got", "gotten" ];
+
+var forEach = require( "lodash/forEach" );
+var includes = require( "lodash/includes" );
+var isEmpty = require( "lodash/isEmpty" );
+var intersection = require( "lodash/intersection" );
+
+/**
+ * Checks whether a participle is directly preceded by a given word.
+ *
+ * @param {Array} precedingWords The array of objects with matches and indices.
+ * @param {number} participleIndex The index of the participle.
+ *
+ * @returns {boolean} Returns true if the participle is preceded by a given word, otherwise returns false.
+ */
+var includesIndex = function( precedingWords, participleIndex ) {
+	if ( ! isEmpty( precedingWords ) ) {
+		var precedingWordsEndIndices = [];
+		forEach( precedingWords, function( precedingWord ) {
+			var precedingWordsEndIndex = precedingWord.index + precedingWord.match.length;
+			precedingWordsEndIndices.push( precedingWordsEndIndex );
+		} );
+		return includes( precedingWordsEndIndices, participleIndex );
+	}
+	return false;
+};
+
+/**
+ * Looks for a specific participle and checks whether if it is directly preceded by a determiner.
+ * If the participle is directly preceded by a determiner it is not a verb, hence the sentence part is not passive.
+ *
+ * @param {string} participle The participle.
+ * @param {string} word The word to check whether it is preceded by a determiner.
+ * @param {string} sentencePart The sentence part to find the participle and word index in.
+ *
+ * @returns {boolean} Returns true if the participle is preceded by a determiner, otherwise returns false.
+ */
+var isPrecededByDeterminer = function( participle, word, sentencePart ) {
+	if ( participle !== word ) {
+		return false;
+	}
+	var indexOfFit = sentencePart.indexOf( word );
+	var determinerMatches = matchDeterminers( sentencePart );
+	return includesIndex( determinerMatches, indexOfFit );
+};
+
+/**
+ * Creates an Participle object for the English language.
+ *
+ * @param {string} participle The participle.
+ * @param {string} sentencePart The sentence part that contains the participle.
+ * @param {string} auxiliary  The auxiliary in the sentence part.
+ * @param {string} type The type of participle found.
+ *
+ * @constructor
+ */
+var EnglishParticiple = function( participle, sentencePart, auxiliary, type ) {
+	Participle.call( this, participle, sentencePart, auxiliary, type );
+	this.isException();
+};
+
+require( "util" ).inherits( EnglishParticiple, Participle );
+
+/**
+ * Sets sentence part passiveness to passive if there is no exception.
+ * @returns {void}
+ */
+EnglishParticiple.prototype.isException = function() {
+	if ( isEmpty( this.getParticiple() ) ) {
+		this.setSentencePartPassiveness( false );
+		return;
+	}
+	var isPassive =
```

Comment:
But you are comparing five things to determine if a sentence is passive. If you already have a function to do this in the Participle object, can't you use that here?